pallets/ddc-clusters/src/lib.rs

```diff
@@ -201,5 +218,36 @@ pub mod pallet {
 
 			Ok(())
 		}
+
+		/// Allow cluster node candidate to chill in the next DDC era.
+		///
+		/// The dispatch origin for this call must be _Signed_ by the controller.
+		#[pallet::weight(10_000)]
+		pub fn fast_chill(origin: OriginFor<T>) -> DispatchResult {
+			let controller = ensure_signed(origin)?;
+
+			let stash = <pallet_ddc_staking::Pallet<T>>::ledger(&controller)
+				.ok_or(<pallet_ddc_staking::Error<T>>::NotController)?
+				.stash;
+			let node = <pallet_ddc_staking::pallet::Nodes<T>>::iter()
```

Comment:
Is this iteration okay in case we have a lot of staking nodes?

Reply:
We can add an additional parameter if we expect frequent invocations of this call. What do you think?